site/app/libraries/Output.php

```diff
@@ -100,10 +100,17 @@ public function getRender() {
         return $this->render;
     }
 
+    /**
+     * @return \Twig\Environment|null
+     */
+    public function getTwig() {
+        return $this->twig;
+    }
```

Comment:
It's generally better to use native type annotations instead of PHPDoc comments if possible.  Native type annotations are enforced at runtime in debug mode, as opposed to PHPDoc comments which are merely comments.

```suggestion
    public function getTwig(): ?\Twig\Environment {
        return $this->twig;
    }
```